content/docs/command-reference/pull.md

```diff
@@ -18,40 +18,42 @@ positional arguments:
 
 ## Description
 
-The `dvc pull` and `dvc push` commands are the means for uploading and
-downloading data to and from remote storage. These commands are analogous to
-`git pull` and `git push`, respectively.
+The `dvc push` and `dvc pull` commands are the means for uploading and
+downloading data to and from remote storage (S3, SSH, GCS, etc.). These commands
+are similar to `git push` and `git pull`, respectively.
 [Data sharing](/doc/use-cases/sharing-data-and-model-files) across environments
 and preserving data versions (input datasets, intermediate results, models,
-[metrics](/doc/command-reference/metrics), etc) remotely (S3, SSH, GCS, etc.)
-are the most common use cases for these commands.
+[metrics](/doc/command-reference/metrics), etc.) remotely are the most common
+use cases for these commands.
 
-The `dvc pull` command allows one to retrieve data from remote storage.
-`dvc pull` has the same effect as running `dvc fetch` and `dvc checkout`
-immediately after.
+`dvc pull` downloads data from [remote storage](/doc/command-reference/remote)
+and places it in the <abbr>workspace</abbr>. It has the same effect as running
+`dvc fetch` and `dvc checkout`.
 
-The default remote is used (see `dvc config core.remote`) unless the `--remote`
+> Note that pulling data does not change any `dvc.yaml` or `.dvc` files, nor
+> does it save any changes to the code, `dvc.lock`, or `.dvc` files (that should
```

Comment:
not sure why would I expect it to save changes in the first place?

Reply:
True. I copied this from pull without editing accordingly. Updated.

> I've also applied similar notes to fetch and commit, but didn't push it in this PR for now. I'll make another small PR soon.